src/components/ProductBarInfoPage.js

```diff
@@ -52,9 +52,9 @@ export const ProductBarInfoPage = () => {
   const classes = useStyles();
   const score = calculateScore(randomInfo);
   let colour;
-  if (score >= 10) {
+  if (score >= 8) {
```

Comment:
```suggestion
  if (score >= 9) {
```

According to our team messenger chat: 0-4 bad, 5-8 ok, 9-12 good

Reply:
Right! I'll change that 💯